index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./styles/index.css" rel="stylesheet"/>
+    <script src="./node_modules/axios/dist/axios.min.js"></script>
+    <script src="./src/index.js"></script>
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <div id="headerSection">
+        <header>Weather report </header>
+        <div id="lovelyCity">For the lovely city of ✨<span id="cityNameHeader">Happyland</span>✨</div>
+    </div>
+    <div class="boxes">
+        <div class="block1">
+            <section>
+                <div id="box1" class="box">
+                    <h3 class="boxHeaders">Temperature</h3>
```

Comment:
This should have been a `h2` since you shouldn't skip from `h1` to `h3`:

```suggestion
                    <h2 class="boxHeaders">Temperature</h2>
```

Remember, `h1`, `h2`, `h3`, etc are about the structure of the document not the styles you want.  You can always override styles in CSS.